ansys/dpf/core/element_descriptor.py

```diff
@@ -64,31 +59,18 @@ def __init__(self, element_id, description, name, shape = None, number_of_corner
         self.is_quadratic = is_quadratic
         
     def __str__(self):
-        txt = 'Element descriptor \n'
-        txt += '-'*18 + '\n'
-        txt += "Enum id (dpf.element_types): "
-        txt += str(self.element_id)
-        txt += "\n"
-        txt += "Element description: "
-        txt += str(self.description)
-        txt += "\n"
-        txt += "Element name (short): "
-        txt += str(self.name)
-        txt += "\n"
-        txt += "Element shape: "
-        txt += str(self.shape)
-        txt += "\n"
-        txt += "Number of corner and mid-side nodes: "
-        txt += str(self.number_of_corner_nodes)
-        txt += str(", ")
-        txt += str(self.number_of_mid_nodes)
-        txt += "\n"
-        txt += "Total number of nodes: "
-        txt += str(self.number_of_nodes)
-        txt += "\n"
-        txt += "Is quadratic element: "
-        txt += str(self.is_quadratic)
-        txt += "\n"
-        return txt
+        lines = []
+        lines.append('Element descriptor')
+        lines.append('-'*18)
+        lines.append("Enum id (dpf.element_types): " + str(self.element_id))
+        lines.append("Element description: " + str(self.description))
+        lines.append("Element name (short): " + str(self.name))
+        lines.append("Element shape: " + str(self.shape))
+        nodes_txt = "Number of corner and mid-side nodes: " + str(self.n_corner_nodes)
+        nodes_txt += str(", ") + str(self.n_mid_nodes)
+        lines.append(nodes_txt)
+        lines.append("Total number of nodes: " + str(self.n_nodes))
+        lines.append("Is quadratic element: " + str(self.is_quadratic))
```

Comment:
You may also want to consider f-strings.  For example:

```py
lines.append("Total number of nodes: " + str(self.n_nodes))
```
becomes
```py
lines.append(f"Total number of nodes: {self.n_nodes}")
```
This does preclude its use within Workbench's IronPython env, so I guess we can't if this will be used there as well.

F-strings are really great, and you can even look into correct formatting of the lines to get the printout consistent.  For this, see some of the work here:
https://github.com/pyvista/pyvista/blob/155b932ced36e4fb5a93061e1dd0c73c820e5494/tests/check_example_coverage.py#L157